install/mysql/plugin_formcreator_empty.sql

```diff
@@ -179,6 +179,8 @@ CREATE TABLE IF NOT EXISTS `glpi_plugin_formcreator_targetchanges` (
   `tag_specifics`              varchar(255) NOT NULL DEFAULT '',
   `category_rule`              int(11) NOT NULL DEFAULT '1',
   `category_question`          int(11) NOT NULL DEFAULT '0',
+  `validation_rule`            int(11) NOT NULL DEFAULT '1',
```

Comment:
I think the columns should be renamed : we don't know if the validation is related to form answer or ticket. In the future, we may have headache to maintain and debug.

- ticket_validation_rule
- ticket_validation_question

Reply:
Would "comonitil_validation_rule" be ok instead ? Since it applies to both change and tickets.

Reply:
good point. Ok for me